gapis/shadertools/cc/spv_manager.cpp

```diff
@@ -112,6 +112,18 @@ void SpvManager::renameViewIndex() {
   }
 }
 
+// Remove all layout(location = ...) qualifiers.
+// For most use cases we should be binding/remapping locations regardless whether they
+// are assigned explicit location or compiler generated one.
+// TODO: Handle separate shader objects - we just hope the game uses consistent block names for now.
+bool SpvManager::removeLayoutLocations() {
```

Comment:
It might be worth renaming any variables that have layout information in a deterministic way. This way you don't have to worry about block name consistency.

layout(location = 1) float foo;
can become
float _f_loc1;